SECURITY.md

```diff
@@ -2,10 +2,17 @@
 
 ## Supported Versions
 
-The latest released version of DAML as per https://github.com/digital-asset/daml/releases is currently being supported with security updates.
+The latest released version of DAML as per https://github.com/digital-asset/daml/releases is
+currently being supported with security updates.
```

Comment:
Should this point to the LATEST file instead? The releases page has many snapshot releases we do not plan to patch in any way, even for security issues.

Reply:
Oh, good point. There are also snapshots in LATEST though; perhaps we should be clear that snapshots are not supported?

Reply:
Or we could link to https://github.com/digital-asset/daml/releases/latest.

Reply:
I would suggest to either link to latest as you proposed or to https://docs.daml.com/support/release-notes.html.

Reply:
I'll go with the docs; seems more "official".